palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/service/PalisadeService.java

```diff
@@ -30,6 +30,11 @@
  */
 public interface PalisadeService extends Service {
 
+    /**
+     * Service name
+     */
+    String NAME = "palisade-service";
```

Comment:
Currently causing a SonarQube error
```suggestion
    public static final String NAME = "palisade-service";
```

Reply:
SonarQube doesn't like this being in an interface. Smell states that constants should be in a class or enum, will move it back to SimplePalisadeService and make it a static constant with the same value

Reply:
So SQ doesn't complain at the other interfaces where it is a static final constant.

Reply:
The other constants are in classes, not interfaces within the Palisade-service module.

Reply:
All service names have been put into an enum.